src/analyses/cfg_dominators.h

```diff
@@ -414,38 +459,51 @@ class cfg_dominators_templatet
   };
 };
 
+/// Dominator tree computation
+/// Follows "A fast algorithm for finding dominators in a flow graph" of
+/// Lengauer and Tarjan. Node indices are 1-based as in the paper, with the
+/// first element (with index 0) of each data structure simply left empty.
 template <class P, class T, bool post_dom>
 void cfg_dominators_templatet<P, T, post_dom>::fixedpointt::fixedpoint(
   P &program)
 {
-  // Dominator Tree according to Lengauer and Tarjan
-  // "A fast algorithm for finding dominators in a flow graph"
-  // This is ununderstandable without reading the paper!
-  // assumption: Vertex indices >= 0 and < cfg.size()
```

Comment:
In hindsight, "ununderstandable" was probably not the right choice of words :slightly_smiling_face: